### PR TITLE
1.18 Release

### PR DESCRIPTION
**Pull request by evilnick**
## Done

- [x] Remove `microk8s.*` and replaced with `microk8s *`
- [x] Add release notes
- [x] Add Windows installer instructions
- [x] Add MacOS installer instructions

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

### CODE REVIEW

**Comment by evilnick** [comment]
Because of the way the page is constructed, there is no way to link directly to the MacOS install on the main page from the docs pages, so I updated both.

The main install instructions:
https://microk8s-io-canonical-web-and-design-pr-234.run.demo.haus/
(click on the MacOS link to see them)

The nicer docs page is here:
https://microk8s-io-canonical-web-and-design-pr-234.run.demo.haus/docs/install-alternatives#macos

**Comment by evilnick** [comment]
Windows changes are done now too, so this is ready

File: index.html [hunk 1]
```diff
@@ -323,47 +323,32 @@ <h3 class="p-stepped-list__title">Turn on standard services</h3>
...
                 <button class="p-code-copyable__action">Copy to clipboard</button>
               </div>
-            </li>
 
+            </li>
             <li class="p-stepped-list__item">
               <h3 class="p-stepped-list__title">Check the status</h3>
 
               <div class="p-code-copyable u-stepped-list-margin">
-                <input class="p-code-copyable__input" aria-label="sudo microk8s.status --wait-ready" value="sudo microk8s.status --wait-ready" readonly="readonly">
```
Comment (evilnick) [inline index.html]: sudo or not sudo? I don't currently have a way of testing

File: docs/install-alternatives.md [hunk 2]
```diff
@@ -13,32 +13,83 @@ notes below.
...
+    ```
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+    ```
+
+1.  **Install MicroK8s**
+
+    Run the command:
+
+    ```
+    brew install ubuntu/microk8s/microk8s
```
Comment (ktsakalozos) [inline docs/install-alternatives.md]: We probably also need a `microk8s install`, right @joedborg ?

File: docs/release-notes.md [hunk 3]
```diff
@@ -6,33 +6,61 @@ permalink: /docs/release-notes
 
 # Release notes
 
-## MicroK8s 1.17 - 11 December 2019
+## MicroK8s 1.18 - XX March 2020
```
Comment (ktsakalozos) [inline docs/release-notes.md]: We should update this, right?

File: docs/install-alternatives.md (hunk 2 above)
Comment (evilnick) [reply docs/install-alternatives.md]: not according to the asciinema cast

File: docs/release-notes.md (hunk 3 above)
Comment (evilnick) [reply docs/release-notes.md]: yes. what date do you want?

**Pull request merged**